build/aca-machine-init.sh

```diff
@@ -97,28 +100,24 @@ echo "6--- installing openvswitch dependancies ---" && \
     make && \
     make install && \
     cp /var/local/git/openvswitch/lib/vconn-provider.h /usr/local/include/openvswitch/vconn-provider.h && \
-    cd /usr/local/bin && \
-    rm ov* && \
```

Comment:
we do want to remove the newly built ovs executables including ovs-vsctl/ovs-ofctl and use the one that already installed in the system. e.g. an openstack setup.

The purpose of cloning and building the openvswitch code is just to get its headers and libraries to compile with ACA.

Reply:
that would make build env not capable to run local test as library aca used could be quite different from upstream ubuntu openvswitch-switch user space tools

Reply:
we don't want to make it a requirement: "setup ACA will replace the OVS on the machine (e.g. compute node in openstack)". ACA talks to openvswitch via a standard openflow protocol, therefore, it will be backward and forward compatible for many releases.

Reply:
ok, will take this out

Reply:
> add back the openvswitch-switch installed at beginning? 

please keep it removed.

> BTW, the ovs make install will install new user space servers at /usr/local/sbin which will shadow the ones installed through apt, should them (ovs-vswitchd, ovsdb-server)be removed as well here?

Good point, please remove them as well.